finance-api/concepts/how-card-payments-work-at-Zettle.md

```diff
@@ -0,0 +1,21 @@
+How card payments work at Zettle
+===
+For every merchant who makes transactions with Zettle, they have an Zettle account that handles those transactions.
+
+As shown in [Figure 1 Zettle card transaction flow](#Zettle-card-transaction-flow), after a merchant makes a transaction with Zettle, the money of the transaction will go through the following phases in sequence:
+1. Recorded in the merchant's Zettle preliminary account:
+ 
+   Zettle checks the money with a third-party acquirer who confirms the transaction.
+   
+2. Recorded in the merchant's Zettle liquid account:
+
+   After the third-party acquirer confirms the transaction, the money is ready to be paid out to the merchant.
+   
+3. Recorded as paid-out: 
+  
+   Zettle reconciles the transaction to make sure that the correct amount of money will be paid out from the merchant's Zettle account to the merchant's bank account. Then the merchant will receive the money.
```

Comment:
I don't get what flow is this. We usually payout everything in the liquid account with a few exceptions.